config/base/crds/kafka.banzaicloud.io_kafkaclusters.yaml

```diff
@@ -12303,6 +12303,17 @@ spec:
                     type: object
                   cruiseControlEndpoint:
                     type: string
+                  cruiseControlOperatonSpec:
+                    description: CruiseControlOperatonSpec specifies the configuration
+                      of the CruiseControlOperaton handling
+                    properties:
+                      ttlSecondsAfterFinished:
+                        description: TTLSecondsAfterFinished when is specified, the
+                          created and succeeded CruiseControlOperation custom resource
+                          will be deleted after the given time elapsed.
```

Comment:
```suggestion
                        description: TTLSecondsAfterFinished specifies when the
                          created and succeeded CruiseControlOperation custom resource
                          will be deleted in seconds. When not specified it will not be deleted.
```

Reply:
I have modified this.